client/tests/unit/analyze-pending.spec.ts

```diff
@@ -1,16 +1,34 @@
 import { createLocalVue, mount } from '@vue/test-utils'
 import Vuetify from 'vuetify'
-import Comp from '@/components/new-request/analyze-pending.vue'
+import AnalyzePending from '@/components/new-request/analyze-pending.vue'
+import newReqModule from '@/store/new-request-module'
 
 const localVue = createLocalVue()
 const vuetify = new Vuetify()
 
 localVue.use(Vuetify)
 
+function setState () {
+  newReqModule.mutateEntityType('CR')
+  newReqModule.mutateLocation('BC')
+  newReqModule.mutateRequestAction('NEW')
+  newReqModule.mutateAnalyzePending(true)
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve('resolved')
+    }, 100)
+  })
+}
+
 describe('analyze-pending.vue', () => {
-  const wrapper = mount(Comp, {
-    localVue,
-    vuetify
+  let wrapper: any
+  beforeEach(async (done) => {
+    await setState()
+    wrapper = mount(AnalyzePending, {
+      localVue,
+      vuetify
+    })
+    done()
```

Comment:
You don't need to call done(). We can chat separately on when done() is needed. (In a nutshell, never. It's better to use async/await.)